src/osd/OSD.cc

```diff
@@ -5942,7 +5943,8 @@ OSDService::ScrubJob::ScrubJob(const spg_t& pg, const utime_t& timestamp, bool m
       sched_time += rand() % (int)(g_conf->osd_scrub_min_interval *
```

Comment:
Why will no load average be considered. In my proposal osd_scrub_max_interval is still used until it is more then pool_scrub_interval, so the user can only make delay more per pool.

Saying that, it may really be confusing for users. Also, it looks we are going to add yet another per pool settings and might need a different way to store/display them (see "Pool setting for recovery priority" thread in ceph-devel). So adding both max and min scrub interval (together with bunch of other settings) might be ok then.